docs/extension.md

```diff
@@ -290,3 +297,97 @@ paths:
                   type: integer
                   format: int32l
 ```
+
+Note: currently `mocking` is incompatible with the `validation` option. If `mocking` is enabled, the validation will be silently disabled.
```

Comment:
I don't think this is good UX. It would probably be better to fail in this case and be explicit as to why

Reply:
Agree, fixed, however it will fail during the configuration apply only if specified on the same x-kusk level - the inheritance is not written for the API webhook. It will fail completely in the manager reconciler.